src/livestreamer/plugins/tvcatchup.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+from livestreamer import PluginError
+from livestreamer.plugin import Plugin
+from livestreamer.plugin.api import http
+from livestreamer.stream import HLSStream
+
+SUCCESS_HTTP_CODES = (200,)
+
+STREAM_URL_FORMAT = "http://tvcatchup.com/stream.php?chan={0}"
+_url_re = re.compile("http://(?:www\.)?tvcatchup.com/watch/(?P<channel_id>[0-9]+)")
+
+
+class TVCatchup(Plugin):
+    def __init__(self, url):
+        Plugin.__init__(self, url)
+        match = _url_re.match(url).groupdict()
+        self.channel_id = match["channel_id"]
+
+    @classmethod
+    def can_handle_url(self, url):
+        print "hi!", url
```

Comment:
No, it's not meant to be there at all... My bad, was testing it before - crappy debug method.